### PR TITLE
Move Ops in relay.op.contrib.*

### PR DESCRIPTION
**Pull request by comaniac**
According to the discussion in #4933, we should move the level 10 ops out of `relay.op.contrib`. 

This PR moves `adaptive_max_pool2d` and `adaptive_avg_pool2d` to `relay.op.nn`; `ndarray_size` to `relay.op.tensor`.

In addition, I keep `relay.op.contrib` as a placeholder for #4933.

cc @zhiics @icemelon9 @kevinthesun

### CODE REVIEW

File: python/tvm/relay/op/nn/_nn.py [hunk 1]
```diff
@@ -620,3 +620,12 @@ def pad_shape_func(attrs, inputs, _):
 reg.register_shape_func("nn.bias_add", False, elemwise_shape_func)
 reg.register_shape_func("nn.softmax", False, elemwise_shape_func)
 reg.register_shape_func("nn.relu", False, elemwise_shape_func)
+
```
Comment (icemelon9) [inline python/tvm/relay/op/nn/_nn.py]: move the following register above the shape func section.

Comment (comaniac) [reply python/tvm/relay/op/nn/_nn.py]: Thanks for suggestion. Updated.

File: python/tvm/relay/op/nn/_nn.py [hunk 2]
```diff
@@ -246,6 +246,15 @@ def legalize_conv2d_transpose(attrs, inputs, types):
...
 
+# adaptive_max_pool2d
+reg.register_schedule("nn.adaptive_max_pool2d", strategy.schedule_adaptive_pool)
+reg.register_pattern("nn.adaptive_max_pool2d", OpPattern.OUT_ELEMWISE_FUSABLE)
+
+
+# adaptive_avg_pool2d
+reg.register_schedule("nn.adaptive_avg_pool2d", strategy.schedule_adaptive_pool)
+reg.register_pattern("nn.adaptive_avg_pool2d", OpPattern.OUT_ELEMWISE_FUSABLE)
+
```
Comment (icemelon9) [inline python/tvm/relay/op/nn/_nn.py]: add one more empty line here

Comment (comaniac) [reply python/tvm/relay/op/nn/_nn.py]: Updated.

**Comment by kevinthesun** [comment]
Thanks @comaniac @icemelon9 @zhiics

**Comment by tqchen** [comment]
@comaniac This PR introduces quite a few doc warnings, please fix them, then we can enable the build checks

**Comment by tqchen** [comment]
see also http://ci.tvm.ai:8080/job/temp-ci-docker-staging/job/ci-stage/18/execution/node/228/log/

**Comment by comaniac** [comment]
Oops. Sorry for that. Will do.

**Pull request merged**